payjoin/src/receive/v1/mod.rs

```diff
@@ -805,20 +807,14 @@ pub(crate) mod test {
                         .unwrap()
                         .require_network(network)
                         .unwrap())
-            })?)
-    }
-
-    #[test]
-    fn can_get_proposal_from_request() {
-        let proposal = proposal_from_test_vector();
-        assert!(proposal.is_ok(), "OriginalPSBT should be a valid request");
+            })
+            .expect("Receiver output should be identified"))
     }
 
-    #[test]
-    fn unchecked_proposal_unlocks_after_checks() {
-        let proposal = proposal_from_test_vector().unwrap();
-        assert_eq!(proposal.psbt_fee_rate().unwrap().to_sat_per_vb_floor(), 2);
-        let payjoin = proposal
+    fn provisional_proposal_from_test_vector(
+        proposal: UncheckedProposal,
+    ) -> Result<ProvisionalProposal, BoxError> {
+        Ok(proposal
```

Comment:
I'd prefer something like:

```suggestion
        Ok(wants_outputs_from_test_vector(proposal).commit_outputs().commit_inputs())
```